addons/event/tests/test_event_mail_schedule.py

```diff
@@ -320,6 +321,58 @@ def test_event_mail_schedule(self):
                 'subject': f"Reminder for {test_event.name}: today",
             })
 
+    @mute_logger('odoo.addons.event.models.event_mail')
+    @users('user_eventmanager')
+    def test_event_mail_schedule_fail_global_composer(self):
+        """ Simulate a fail during composer usage e.g. invalid field path, template
+        / model change, ... to check defensive behavior """
+        cron = self.env.ref("event.event_mail_scheduler").sudo()
+        before_scheduler = self.test_event.event_mail_ids.filtered(lambda s: s.interval_type == "before_event")
+        self._create_registrations(self.test_event, 2)
+
+        def _patched_send_mail(self, *args, **kwargs):
+            raise exceptions.ValidationError('Some error')
+
+        with patch.object(type(self.env["mail.template"]), "send_mail_batch", _patched_send_mail):
+            with self.mock_datetime_and_now(self.reference_now + relativedelta(days=3)), \
+                 self.mock_mail_gateway():
+                cron.method_direct_trigger()
+        self.assertFalse(before_scheduler.mail_done)
```

Comment:
Unclear to me why this one is False and the one below is True when they both fail?